README.md

```diff
@@ -486,3 +486,13 @@ curl -X GET -H "$(oauth2l header --json build/exploded-api/WEB-INF/sa-key.json u
 # If you get 401 errors, you may need to clear your token cache.
 oauth2l reset
 ```
+
+### Generating survey_question_map.csv
+survey_question_map.csv is used to identify the order of appearance of the questions in actual survey pdfs.
+1. Id column is incremental and specifies the order of the questions in each survey.(Used to display the questions in the same order in databrowser)
+2. survey_concept_id holds the concept id of the survey. (can be fetched from the survey_module))
+3. question_concept_id holds the concept id of each question. (can be fetched from the survey pdf)
+4. path specifies the origin id of each question. If the question is the sub-question, then the id of the main question would be the path.
+5. is_main is the boolean field which holds 1 in the case of main questions and 0 in case the question is the sub question.
+* NOTE: This file for now is generated manually and in case new questions come in any of the survey, specific row should be added to this file to avoid missing out count generation for the new ones.
```

Comment:
What will happen if we forget to update this? Would there be Exceptions? Would the data look incorrect?

Reply:
The achilles_results would not have counts in there and the question qould not be displayed. Added that info in there.